src/github/client/github-queries.ts

```diff
@@ -362,46 +362,9 @@ export type getDeploymentsResponse = {
 	}
 };
 
-export const getDeploymentsQueryByCreatedAtDesc = `query ($owner: String!, $repo: String!, $per_page: Int!, $cursor: String) {
-  repository(owner: $owner, name: $repo){
-    deployments(first: $per_page, after: $cursor, orderBy: { direction: DESC, field: CREATED_AT }) {
-      edges {
-        cursor
-        node {
-					createdAt
-          repository {
-            id: databaseId
-            node_id: id
-            name
-            owner {
-              login
-            }
-          }
-          databaseId
-          commitOid
-          task
-          ref {
-            name
-            id
-          }
-          environment
-          description
-          latestStatus {
-            environmentUrl
-            logUrl
-            state
-            id
-            updatedAt
-          }
-        }
-      }
-    }
-  }
-}`;
-
 export const getDeploymentsQuery = `query ($owner: String!, $repo: String!, $per_page: Int!, $cursor: String) {
   repository(owner: $owner, name: $repo){
-    deployments(first: $per_page, after: $cursor) {
+    deployments(first: $per_page, after: $cursor, orderBy: { direction: DESC, field: CREATED_AT }) {
```

Comment:
What's the rollout strategy for this?

Reply:
No strategy, deploy, hit the api.
Or I could add a ff flag. So deploy, turn on ff, hit the api.
Thoughts?

Reply:
Right, so you would:
1. get the IDs of the records that are in sync now (by looking at updatedAt?)
2. trigger endpoint with these IDs?

At which point of time would you collect them, because there could be race-conditions:
- you turned on FF
- a sync has stopped // <-- this will be broken
- you triggered endpoint
?

I was thinking of maybe doing something like:
- collect all active IDs before deploy
- collect all active IDs after deploy
- merge them together - these need to be resynced
- drop statuses
- trigger partial resync for the affected installationIds

wdyt?

Reply:
Make sense. 
I've updated the PR to allow RepoSyncStates with specified ids to be reset for above operations.